Html/Home/index.html

```diff
@@ -55,19 +76,99 @@
   <main>
     <!-- banner -->  
     <section class="section-banner-slide">
-      <h1>NEW COLLECTION</h1>
+      <h1>NEW <br> COLLECTION</h1>
```

Comment:
O `<br>` funciona mas fica engessado , uma sugestão seria você usar assim:
```suggestion
      <h1>NEW <span>COLLECTION</span></h1>
```
Aí você consegue customizar o `span` mais facilmente se necessário

Reply:
resolvi colocando     word-wrap: break-word;